command/tests/test_identify.py

```diff
@@ -0,0 +1,28 @@
+from unittest import TestCase
+from nose.tools import nottest
+import command.identify as id
+
+class TestGetOperatorArtifactType(TestCase):
```

Comment:
If I can, I suggest to use pytest instead of nose, this whole test can be simplified in pytest using parametrize fixture

```python
@pytest.mark.parametrize('fname,expected', [
    ("command/tests/test_files/csv.yaml", "clusterServiceVersions"),
    ("command/tests/test_files/crd.yaml", "customResourceDefinitions"),
   ....
])
def test_get_operator_artifact_type(fname, expected):
    with open(fname) as f:
        yaml = f.read()
   assert getOperatorArtifactType(yaml) == expected
```

Reply:
Great point - I will address this in the next commit.